urdf/src/model.cpp

```diff
@@ -136,12 +133,12 @@ bool Model::initXml(TiXmlElement *robot_xml)
   return Model::initString(ss.str());
 }
 
-bool Model::initString(const std::string& xml_string)
+bool Model::initString(const std::string & xml_string)
 {
   urdf::ModelInterfaceSharedPtr model;
 
   // necessary for COLLADA compatibility
-  if( IsColladaData(xml_string) ) {
+  if (IsColladaData(xml_string) ) {
```

Comment:
I think the whitespace after the condition (between the 2 closing prenthesis) should be removed too